On macos 10.15:

```
  CCLD     bz2.la
ld: unknown option: -z
clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
make[4]: *** [Makefile:741: heif.la] Error 1
make[4]: *** Waiting for unfinished jobs....
  CCLD     lzma.la
ld: unknown option: -z
clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
make[4]: *** [Makefile:765: svg.la] Error 1
make[4]: Leaving directory '/private/tmp/nix-build-imlib2-1.8.1.drv-0/imlib2-1.8.1/src/modules/loaders'
make[3]: *** [Makefile:401: all-recursive] Error 1
make[3]: Leaving directory '/private/tmp/nix-build-imlib2-1.8.1.drv-0/imlib2-1.8.1/src/modules'
make[2]: *** [Makefile:401: all-recursive] Error 1
make[2]: Leaving directory '/private/tmp/nix-build-imlib2-1.8.1.drv-0/imlib2-1.8.1/src'
make[1]: *** [Makefile:523: all-recursive] Error 1
make[1]: Leaving directory '/private/tmp/nix-build-imlib2-1.8.1.drv-0/imlib2-1.8.1'
make: *** [Makefile:430: all] Error 2
builder for '/nix/store/9h4qw6713796w2jk1adj4m7qqm8f0b27-imlib2-1.8.1.drv' failed with exit code 2
```

--- a/pkgs/development/libraries/imlib2/default.nix
+++ b/pkgs/development/libraries/imlib2/default.nix
@@ -2,7 +2,7 @@
 # Image file formats
 , libjpeg, libtiff, giflib, libpng, libwebp
 # imlib2 can load images from ID3 tags.
-, libid3tag
+, libid3tag, librsvg, libheif
 , freetype , bzip2, pkg-config
 , x11Support ? true, xlibsWrapper ? null
 }:
@@ -21,8 +21,11 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     libjpeg libtiff giflib libpng libwebp
-    bzip2 freetype libid3tag
-  ] ++ optional x11Support xlibsWrapper;
+    bzip2 freetype libid3tag libheif
+  ] ++ optional x11Support xlibsWrapper
+  # Compilation error on Darwin with librsvg. For more information see:
+  # https://github.com/NixOS/nixpkgs/pull/166452#issuecomment-1090725613
+  ++ optional (!stdenv.isDarwin) librsvg;
 
   nativeBuildInputs = [ pkg-config ];
 

